src/plugins/mer/mervirtualboxmanager.cpp

```diff
@@ -581,7 +703,48 @@ QStringList listedVirtualMachines(const QString &output)
     return vms;
 }
 
-VirtualMachineInfo virtualMachineInfoFromOutput(const QString &output)
+void fetchVdiInfo(VirtualMachineInfo &virtualMachineInfo)
```

Comment:
Please move fetchVdiInfo and vdiInfoFromOutput after virtualMachineInfoFromOutput.